Testscripts/Linux/customKernelInstall.sh

```diff
@@ -224,9 +224,10 @@ function InstallKernel() {
         echo "deb http://archive.ubuntu.com/ubuntu/ ${release}-proposed restricted main multiverse universe" >> /etc/apt/sources.list
         rm -rf /etc/apt/preferences.d/proposed-updates
         LogMsg "Installing linux-azure kernel from $release proposed repository."
-        apt clean all
-        apt -y update >> $LOG_FILE 2>&1
-        apt install -yq linux-azure/"$release" >> $LOG_FILE 2>&1
+        apt-get clean all
+        apt-get -y update >> $LOG_FILE 2>&1
+        CheckInstallLockUbuntu
```

Comment:
There is a situation here, that there is no dpkg process checked when invoke CheckInstallLockUbuntu, after CheckInstallLockUbuntu invoked, the process appear, then we will still hit lock issue when install package.

Reply:
you can easily add a retry on failure, if needed. I think maybe 3 retries with 1 s timeout?